dev/TabView/TabView.idl

```diff
@@ -37,14 +37,21 @@ unsealed runtimeclass TabView : Windows.UI.Xaml.Controls.ListView
     Object RightCustomContent{ get; set; };
     Windows.UI.Xaml.DataTemplate RightCustomContentTemplate{ get; set; };
 
+    [MUX_DEFAULT_VALUE("true")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean IsAddButtonVisible{ get; set; };
+
     event Windows.Foundation.TypedEventHandler<TabView, TabViewTabClosingEventArgs> TabClosing;
 
+    event Windows.Foundation.TypedEventHandler<TabView, Object> AddButtonClick;
```

Comment:
>AddButtonClick [](start = 64, length = 14)

I'd recommend changing the API name to something like RequestAddTab or something else that's not specific to Button or Click - since the button and the fact that a button was clicked feels more like an implementation detail of the default template.

Reply:
I just called everything what we agreed on temporarily in spec review. TabView will go through API review soon and we'll be renaming everything I'm sure.